slope/index.js

```diff
@@ -130,6 +132,7 @@ parseData.fromCSV('./data.csv', dateStructure).then(({ seriesNames, groupNames,
         myChart.dotRadius(currentFrame.rem() * 0.3);
 
         // specifics based on frame type
+        myChart.setPalette(setColourPalette);
         myChart.colourPalette(frameName, groupNames); // set colour palette
```

Comment:
Ah, I see now what you're doing — setPalette takes a boolean, and short-circuits colourPalette. Maybe it should just be passed to `.colourPalette()` as an argument?

Also, is it too late to rename "colourPalette" to "colorPalette"?